src/cli.coffee

```diff
@@ -15,18 +9,25 @@ options = docopt """
   """
 
 
-generateConfig = ->
-  break for extension in extensions when options[extension]
+{docopt} = require 'docopt'
+options = docopt usage
+
+
+generateConfig = (extension) ->
+  asyncHandlers = require 'async-handlers'
+  fsExtra = require 'fs-extra'
```

Comment:
This was another debate I was having. Should requires used in a scope all be at the top or only required when they are used